src/test/java/com/microsoft/sqlserver/jdbc/SQLServerConnectionTest.java

```diff
@@ -1030,4 +1031,64 @@ public void testServerNameField() throws SQLException {
             assertTrue(e.getMessage().matches(TestUtils.formatErrorMsg("R_errorServerName")));
         }
     }
+
+    @Tag(Constants.xSQLv11)
+    @Tag(Constants.xSQLv12)
+    @Tag(Constants.xSQLv14)
+    @Tag(Constants.xSQLv15)
+    @Tag(Constants.xAzureSQLDW)
+    @Tag(Constants.reqExternalSetup)
+    @Test
+    public void testDSPooledConnectionAccessTokenCallback() throws Exception {
+        SQLServerConnectionPoolDataSource ds = new SQLServerConnectionPoolDataSource();
+        AbstractTest.updateDataSource(connectionString, ds);
+        ds.setUser("");
+        ds.setPassword("");

```

Comment:
is this necessary?

Reply:
Yep, the connection string on the pipelines by default have the user and password set. Access token call back doesn't require a user and password. If user and password is set and access token call back is used, it will fail/throw exception.

Reply:
I see, so this is just to get around setup in our test lab, not that it's required interface to use this? If that's the case, then why not just use TestUtils.addorOverrideProperty as in other tests?

Reply:
I'll switch to using that